CHANGELOG.md

```diff
@@ -12,6 +12,16 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 	https://changelog.md/
 -->
 
+
+## v2.0.0 (WIP)
+
+- Refines api call order to getToken -> getProvider -> GitHub Operations instead of the previous getProvider -> getToken
+  -> GitHub operations.  (#10)
+- Adds AvatarURL and GitURL to the putProject POST. (#10)
+- Updates go from 1.13 -> 1.16. (#10)
+- Updates alpine base image from 3.13.4 -> 3.13.5. (#10)
+- Switches from POST to PATCH calls for tokens to eliminate entry duplication. (#10)
+
```

Comment:
It dosent seem to integrate with the jetbrains ides quite yet. Maybe a me project for a slow weekend? The cli works well but is easy to forget. 
I considered the possibility of adding linting checks through the github actions. Maybe this could be a first check?

Reply:
Codacy is doing that already through GitHub checks. I've sometimes cited the Codacy errors